ci/gpu/build.sh

```diff
@@ -90,6 +90,10 @@ fi
 # https://docs.rapids.ai/maintainers/depmgmt/
 # gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
 # gpuci_mamba_retry install -y "your-pkg=1.0.0"
+gpuci_conda_retry remove -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
+      --force rapids-build-env rapids-notebook-env
+gpuci_mamba_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
+      treelite=3.1.0
```

Comment:
This script is no longer in use by GitHub Actions.

We will be removing it from the repository in the coming weeks.

This change will have no effect on the current CI run.

If the current CI run passes, there is no need to push any additional commits to remove this since it will ultimately be deleted in the future anyway. Unless you want to remove it. Up to you.

Reply:
> dependencies from the integration repository are no longer used in CI.

Very nice!